layouts/partials/header.html

```diff
@@ -75,13 +75,20 @@
               </div>
             </div>
             {{ end }}
-
+        <div id="head-tags">
+        {{ partial "tags.html" . }}
+        </div>
         {{ if .Params.chapter }}
           <div id="chapter">
         {{ end }}
         <div id="body-inner">
           {{if and (not .IsHome) (not .Params.chapter) }}
-            <h1>{{.Title}}</h1>
+            <h1>
+              {{ if eq .Kind "taxonomy" }}
+                {{.Kind}} :
```

Comment:
English doesn't have spaces before colons. Not sure how to handle this :thinking: .

Reply:
I found no other usage of colons in the theme, except for the title `<title>{{ .Title }} :: {{ .Site.Title }}</title>`. Maybe we can use double quotes, or some kind of emphasis in order to get rid of the colon? 

A good solution could be to use the  `lang` attribute of the `html` tag, and CSS. Wrapping the colon in a span `<span class="colon">:</span>` could allow to use something like: 
```css
html[lang="fr"] .colon{
    padding-left:0.2em;
}
```
But even with the `fr-fr` language code in `config.toml`, I did not found a way to change the `lang` attribute in html (which is always `en`).

Reply:
I'd say `::` is good enough. Pretty sure I've already seen it somewhere. @matalo33 what do you think?